runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -325,6 +325,20 @@ public static DataflowRunner fromOptions(PipelineOptions options) {
                 new StreamingFnApiCreateOverrideFactory()));
       }
       overridesBuilder
+          // Support Splittable DoFn for now only in streaming mode.
+          // The order of the following overrides is important because they are applied in order.
+
+          // Dataflow runner does not expand single-output ParDo into multi-output ParDo
+          // by default. However, we *do* want to do that for both single-output splittable ParDo.
+          .add(
+              PTransformOverride.of(
+                  PTransformMatchers.splittableParDoSingle(),
+                  new ReflectiveOneToOneOverrideFactory(
+                      SplittableParDoOverrides.ParDoSingleViaMulti.class, this)))
+          .add(
+              PTransformOverride.of(
+                  PTransformMatchers.splittableParDoMulti(),
```

Comment:
Yeah, SplittableParDo handles only multi-output pardos, so I think we need both overrides - the one for reducing single to multi, and replacing multi with SPD.